spec/integration/base_spec.rb

```diff
@@ -135,6 +135,25 @@ class Book < ActiveFedora::Base
     end
   end
   
+  describe "#apply_schema" do
+    before do
+      class ExampleSchema < ActiveTriples::Schema
+        property :title, predicate: RDF::DC.title
+      end
+      class ExampleBase < ActiveFedora::Base
+        apply_schema ExampleSchema
+      end
+    end
+    after do
+      Object.send(:remove_const, :ExampleSchema)
+      Object.send(:remove_const, :ExampleBase)
+    end
+    let(:obj) { ExampleBase.new }
+    it "should configure properties and solrize them" do
+      obj.title = ["Test"]
+      expect(obj.to_solr[ActiveFedora::SolrQueryBuilder.solr_name("title", :symbol)]).to eq ["Test"]
```

Comment:
It looks like it's indexing every property even if no indexing hint has been supplied?  I'm not sure that's desired.